Building DAG of jobs...
Using shell: /usr/bin/bash
Provided cores: 2
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	1	blast
	1

rule blast:
    input: /scr1/users/leej39/USDA_RUN22/blast_against_contigs, /scr1/users/leej39/USDA_RUN22/R1_FASTA/Neg.1_R1.fasta
    output: /scr1/users/leej39/USDA_RUN22/blast_against_contigs/Neg.1_against_D12.BAF_genes_nucl.fa.blastout
    jobid: 0
    wildcards: sample=Neg.1, contig_name=D12.BAF_genes_nucl.fa


        mkdir -p /scr1/users/leej39/USDA_RUN22/blast_against_contigs
        blastn -query /scr1/users/leej39/USDA_RUN22/R1_FASTA/Neg.1_R1.fasta -evalue 1e-5 -outfmt 6         -db /scr1/users/leej39/USDA_RUN22/RUN12_blastdb/D12.BAF_genes_nucl.fa
        
Error in rule blast:
    jobid: 0
    output: /scr1/users/leej39/USDA_RUN22/blast_against_contigs/Neg.1_against_D12.BAF_genes_nucl.fa.blastout

RuleException:
CalledProcessError in line 29 of /home/leej39/RNAseq_analysis/Snakefile:
Command ' set -euo pipefail;  
        mkdir -p /scr1/users/leej39/USDA_RUN22/blast_against_contigs
        blastn -query /scr1/users/leej39/USDA_RUN22/R1_FASTA/Neg.1_R1.fasta -evalue 1e-5 -outfmt 6         -db /scr1/users/leej39/USDA_RUN22/RUN12_blastdb/D12.BAF_genes_nucl.fa ' returned non-zero exit status 127.
  File "/home/leej39/RNAseq_analysis/Snakefile", line 29, in __rule_blast
  File "/home/leej39/miniconda3/lib/python3.6/concurrent/futures/thread.py", line 56, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /home/leej39/RNAseq_analysis/.snakemake/log/2018-12-18T124532.113278.snakemake.log
